src/components/RepoSubmission.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { FC, useState } from 'react'
+
+
+const RepoSubmission: FC = ()  => {
+  
+  const [buttonPlaceHolder, setButtonPlaceHolder] = useState<string>("Submit repo?")
+  const [isFormOpen, setIsFormOpen] = useState<boolean>(false)
+  const [isSubmissionInProcess, setIsSubmissionInProcess] = useState<boolean>(false)
+  const [submitted, setSubmitted] = useState<boolean>(false)
+  const [input, setInput] = useState("")
+
+  
+  
+  const saveDataToDataBase = (url: string):void => {
```

Comment:
console log the url to remove the error.

Reply:
url is now "repoUrl" and is now logged on the console.